### PR TITLE
Invalid DOM Attributes

### LINKED ISSUE #2420: Invalid DOM Properties
## Summary

React is displaying a warning about invalid properties. This is related to the CSS-in-JS update to components. 

These warnings do not break things but indicate that we are violating React convention.

## Steps to reproduce

Deploy the dev version of the app. Navigate to a page with a form, view the browser console.

## Unresolved questions

N/A

### PR DESCRIPTION
**Pull request by justinr86**
Fixes #2420 

# Description

Updated properties to follow React conventions.

# Any new packages installed?

N/A

# Required followup work

N/A

# Checklist:

- [ ] I have updated the azurescript.sh with any **new environment variables** and added them to the appsettings
- [ ] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [ ] I have added and needed tests for my changes (in particular for new screens)
- [ ] I have added a comment to any confusing code
- [ ] I have added documentation to any modified front-end code. (Or added missing documentation)

### CODE REVIEW

File: f2/src/components/formik/datePicker/index.js [hunk 1]
```diff
@@ -79,8 +79,8 @@ const DateEntry = ({ field, form, ...props }) => {
       <Form.Group controlId={props.id}>
         <DateLabel>{props.label}</DateLabel>
         <DateInput
-          hasError={error}
-          dateType={dateType}
+          haserror={error}
+          datetype={dateType}
           type="text"
           {...field}
           maxLength={length}
```
Comment (ying-chen-cds) [inline f2/src/components/formik/datePicker/index.js:86]: If we need to change all properties to lower cases, how about "maxLength" here?

Comment (justinr86) [reply f2/src/components/formik/datePicker/index.js:86]: maxLength is a valid DOM property. The problem here (I should have mentioned in the description) is that Styled-System and Emotion are causing all props to be passed to the rendered element. 

React syntax dictates that DOM properties need to be declared in camel case. To include custom properties they need to be written in lower case.

Comment (justinr86) [reply f2/src/components/formik/datePicker/index.js:86]: https://reactjs.org/docs/introducing-jsx.html#specifying-attributes-with-jsx

**Review by DianeLiu2019 — APPROVED** [review]
all the warning about invalid properties is gone now

File: f2/src/components/formik/datePicker/index.js (hunk 1 above)
Comment (ying-chen-cds) [reply f2/src/components/formik/datePicker/index.js:86]: That makes sense. Thanks, Justin!

**Pull request merged**